app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] After signing up a user and clicking their confirmation link in the email the page it redirects to raises an error. Kindly fix it.

  ![image](https://github.com/Arch-Noize/rorblog/assets/81468675/85acb25e-4170-4d08-8fd2-b958f2396655)